extensions_build_config.bzl

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/envoyproxy/envoy/blob/master/bazel/README.md for details on how this system works.
```

Comment:
The link here provides an avalanche of information.  Do you have objections to using this link instead, which I think is to the more relevant section?
https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#disabling-extensions